src/Views/InstalledView.vala

```diff
@@ -67,22 +67,12 @@ public class AppCenter.Views.InstalledView : View {
 
         yield refresh_mutex.lock ();
 
+        refresh_cancellable.reset ();
+
         unowned AppCenterCore.Client client = AppCenterCore.Client.get_default ();
 
         var installed_apps = yield client.get_installed_applications (refresh_cancellable);
 
-        // To gain some speedup of the installed applications list, update the state of the packages ahead
-        // of adding them to the list. Otherwise, AppCenter will query each one in turn to get the latest
-        // verison string from PackageKit. The update_multiple_package state method ignores packages that
-        // aren't PackageKit packages, so this is safe to run on the whole list, even if it includes Flatpaks
-        if (AppCenterCore.PackageKitBackend.supports_parallel_package_queries && installed_apps != null) {
-            try {
-                yield AppCenterCore.PackageKitBackend.get_default ().update_multiple_package_state (installed_apps);
```

Comment:
We can drop this call to update the state of all of the packages, as the `get_installed_applications` method already sets all of the required fields on the package.